impeller/toolkit/android/README.md

```diff
@@ -0,0 +1,4 @@
+Android Toolkit
+===============
+
+Type-safe managed wrappers around Android objects vended by the NDK. Does not require linking to libandroid.so.
```

Comment:
Nit: please wrap at 80 chars.

Consider explaining that this works by dynamic runtime lookup here, and why (so that you can build targetting older API levels while still taking advantage of newer features)

Reply:
Done.